tests/php/FixFilePermissionsTaskTest.yml

```diff
@@ -0,0 +1,31 @@
+SilverStripe\Assets\Folder:
+  parent:
+    Name: ParentFolder
+    CanViewType: NULL
+  subfolder:
+    Name: SubFolder
+    Parent: =>SilverStripe\Assets\Folder.parent
+SilverStripe\Assets\Image:
+  image1:
+    Name: myimage.jpg
+    CanViewType: LoggedInUsers
```

Comment:
Only folders can have `CanViewType`, at least when they're using https://github.com/silverstripe/silverstripe-secureassets. Files can't. While there could be other ways in which files have data in this column, it's not in scope here - we're only fixing something which a supported module broke, and then had downstream effects on core file migrations.